### PR TITLE
docs: adds usage info for reset command

### PR DESCRIPTION
**Pull request by HarshaVardhanNakkina**
- added usage info for reset command
- added a link in the table of contents too

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,6 +226,29 @@ You can pass an image URL as an option to `cross-post run` using `--image-url`.
 
 You can pass the option `--ignore-image` to `cross-post run` and the article will be published without an image.
 
+##### Reset Configuration Values
+
+you can reset configuration values for each platform like this
+
+```cross-post config reset <platform name>```
```
Comment (shahednasser) [inline README.md:233]: You should add a new line after the first 3 backticks and before the last 3 backticks. Also, you should add the language (in this case `bash`) after the first 3 backticks.

Please apply that to all code blocks

File: README.md [hunk 2]
```diff
@@ -225,6 +226,29 @@ You can pass an image URL as an option to `cross-post run` using `--image-url`.
 
 You can pass the option `--ignore-image` to `cross-post run` and the article will be published without an image.
 
+##### Reset Configuration Values
```
Comment (shahednasser) [inline README.md:229]: Change this to `####`

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your work @HarshaVardhanNakkina I've added some comments below

**Comment by HarshaVardhanNakkina** [comment]
sorry for the delay, I was busy with my work

File: README.md (hunk 1 above)
Comment (HarshaVardhanNakkina) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (HarshaVardhanNakkina) [reply README.md]: done

**Comment by shahednasser** [comment]
Thank you @HarshaVardhanNakkina for your work!

**Pull request merged**